src/v/kafka/server/handlers/topics/types.cc

```diff
@@ -159,6 +159,13 @@ to_cluster_type(const creatable_topic& t) {
     if (cfg.properties.read_replica_bucket.has_value()) {
         cfg.properties.read_replica = true;
     }
+
+    cfg.properties.retention_local_target_bytes = get_tristate_value<size_t>(
```

Comment:
please adjust the `from_cluster_type` in `types.cc`

Reply:
Nice catch.